src/storage/context/StorageExpressionContext.cpp

```diff
@@ -24,7 +25,8 @@ Value StorageExpressionContext::readValue(const std::string& propName) const {
         // read null value
         auto nullType = value.getNull();
         if (nullType == NullType::UNKNOWN_PROP && field->hasDefault()) {
-            return field->defaultValue();
+            DefaultValueContext expCtx;
+            return Value(Expression::eval(field->defaultValue(), expCtx));
```

Comment:
A question, What is the purpose of using this DefaultValueContext? if the `hasDefault()`  just return ConstantExpression. so can we using `StorageExpressionContext` direct？because the `expCtx` is unused in eval method of ConstantExpression class.

Reply:
I think StorageExpressionContext is too redundant to the default value.